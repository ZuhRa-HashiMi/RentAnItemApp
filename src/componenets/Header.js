import React from 'react';
import { Link } from 'react-router-dom';
import '../App.scss'
import Navbar from './Navbar'


export default function Header() {
  return (
    <header className="header">
        <Navbar />

      <div className="header-content mt-5">
        <h1 className="header-content__title">Find housing anywhere</h1>
        <p className="header-content__desc">
          The best offers for you at any point of your journey.
          Classy rides that meets your taste.
        </p>
        <p>
          <Link className="btn btn-oval" to="/register">Hire Now</Link>
        </p>
      </div>
    </header>
  );
}
