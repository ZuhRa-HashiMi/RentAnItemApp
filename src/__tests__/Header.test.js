import React from 'react';
import renderer, { act } from 'react-test-renderer';
import Header from '../componenets/Header';
import { MemoryRouter } from 'react-router';
import store from '../redux/configureStore';
import { Provider } from 'react-redux';

it('renders Footer correctly', () => {
  let tree;
  act(() => {
    tree = renderer.create(
      <React.StrictMode>
        <Provider store={store}>
          <MemoryRouter initialEntries={['/']}>
            <Header />
          </MemoryRouter>
        </Provider>
      </React.StrictMode>,
    );
  });
  expect(tree).toMatchSnapshot();
});
