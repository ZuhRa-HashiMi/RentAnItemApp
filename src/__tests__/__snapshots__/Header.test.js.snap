// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<header
  className="header"
>
  <nav
    className="custom-navbar navbar navbar-expand-sm navbar-light"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      <img
        alt="circle"
        className="logo"
        src="logo-fotor-bg-remover-2023051812511.png"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "2rem",
          "padding": "0.5rem",
        }
      }
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      />
      <div
        className="navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link white "
          href="/login"
          onClick={[Function]}
          style={
            Object {
              "fontSize": "1.3rem",
              "fontWeight": "550",
            }
          }
        >
          SIGN IN
        </a>
        <a
          className="nav-link"
          href="/register"
          onClick={[Function]}
          style={
            Object {
              "fontSize": "1.3rem",
              "fontWeight": "500",
            }
          }
        >
          <span
            className="btn-menu"
          >
            SIGN UP
          </span>
        </a>
      </div>
    </div>
  </nav>
  <div
    className="header-content mt-5"
  >
    <h1
      className="header-content__title"
    >
      Finding house anywhere
    </h1>
    <p
      className="header-content__desc"
    >
      The best offers for you at any point of your journey. where ever you are, feel yourself like at your own home.
    </p>
    <p>
      <a
        className="btn btn-oval"
        href="/register"
        onClick={[Function]}
      >
        Hire Now
      </a>
    </p>
  </div>
</header>
`;
